src/localize.js

```diff
@@ -125,20 +125,58 @@ const strategies = {
   },
 };
 
+// Reference for ratios https://www.w3.org/International/articles/article-text-size
+const expansionRatio = {
+  10: 2.5,
+  20: 1.9,
+  30: 1.7,
+  40: 1.5,
+  50: 1.5,
+  60: 1.6,
+  70: 1.6,
+  over70: 1.3,
+};
+
+const isAlpha = character => {
+  return /[a-zA-Z]/.test(character);
+};
+
+const textExpander = (string, numDuplicateChars = 0, expansionRatio = 1) => {
+  let expandedText = '';
+  let count = 0;
+  const repeatTimes = Math.ceil(expansionRatio);
+
+  for (const c of string) {
+    if (isAlpha(c) && count < numDuplicateChars) {
```

Comment:
Looks like this will not duplicate characters uniformly across the string as was roughly true before when duplicating the common vowels in English. For example, if we would duplicate the string `ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz` it would only duplicate the first `Math.ceil(strLen * ratio) - strLen` characters. Maybe this is ok, but if we are breaking the uniformity of duplication in the string, I think we should start duplicating from the end rather than the start. This would make it work better with text that truncates, as the start of the string would be more legible (no duplication) but the end, that could potentially truncate, would have most of the duplication

Reply:
Can you give an example as I don't understand what do we gain here